widgets/qt_pisces_main_window.py

```diff
@@ -2,106 +2,131 @@
-        left_window_layout.addWidget(qt_custom_decorations.LineBreak())
-
-        left_window_layout.addWidget(qt_textbox_and_slider.InitializeSliderTextB(None, "num_samples", 0, 100, int, 1, 20))
-        left_window_layout.addWidget(qt_custom_decorations.LineBreak())
-
-        left_window_layout.addWidget(qt_textbox_and_slider.InitializeSliderTextB(None, "offset_view1", 0, 3180, float, 0.1, 1550))
-        left_window_layout.addWidget(qt_custom_decorations.LineBreak())
-
-        left_window_layout.addWidget(qt_textbox_and_slider.InitializeSliderTextB(None, "offset_view2", 0, 3180, float, 0.1, 1650))
-        left_window_layout.addWidget(qt_custom_decorations.LineBreak())
-
-        left_window_layout.addWidget(qt_textbox_and_slider.InitializeSliderTextB(None, "view1_galvo1", -10, 10, float, 0.01, 4.2))
-        left_window_layout.addWidget(qt_custom_decorations.LineBreak())
-
-        left_window_layout.addWidget(qt_textbox_and_slider.InitializeSliderTextB(None, "view1_galvo2", -10, 10, float, 0.01, -4.08))
-        left_window_layout.addWidget(qt_custom_decorations.LineBreak())
-
-        left_window_layout.addWidget(qt_textbox_and_slider.InitializeSliderTextB(None, "view2_galvo1", -10, 10, float, 0.01, -4.37))
-        left_window_layout.addWidget(qt_custom_decorations.LineBreak())
-
-        left_window_layout.addWidget(qt_textbox_and_slider.InitializeSliderTextB(None, "view2_galvo2", -10, 10, float, 0.01, 3.66))
-        left_window_layout.addWidget(qt_custom_decorations.LineBreak())
-
-        left_window_layout.addWidget(qt_textbox_and_slider.InitializeSliderTextB(None, "stripe_reduction_range", 0, 10, float, 0.01, 0.1))
-        left_window_layout.addWidget(qt_custom_decorations.LineBreak())
-
-        left_window_layout.addWidget(qt_textbox_and_slider.InitializeSliderTextB(None, "stripe_reduction_offset", -10, 10, float, 0.01, 0.58))
-        left_window_layout.addWidget(qt_custom_decorations.LineBreak())
-
-        #  create container for left_window_layout that can be contained by scroll bar widget
-        left_widget = QWidget()  # QWidget() acts as a container for the laid out widgets - base class for all widgets
-        left_widget.setLayout(left_window_layout)
-
-        #  add scroll bar area that contains left window widget
-        left_scroll_bar = QScrollArea()
-        left_scroll_bar.setMaximumWidth(350)
-        left_scroll_bar.setVerticalScrollBarPolicy(Qt.ScrollBarAlwaysOn)
-        left_scroll_bar.setHorizontalScrollBarPolicy(Qt.ScrollBarAlwaysOff)
-        left_scroll_bar.setWidgetResizable(False)
-        left_scroll_bar.setStyleSheet("background : lightgray;")
-        left_scroll_bar.setWidget(left_widget)
-
-        # add left and right window components to main window (left is a widget due to scroll bar)
-        window_layout.addWidget(left_scroll_bar)
-        window_layout.addLayout(right_window_layout)
+        # Left window
+        self.left_window_layout.setAlignment(Qt.AlignTop)
```

Comment:
Left half of the window possibly can be refactored into a separate file as a single widget, what do you think @ljjh20